common/djangoapps/credo_modules/migrations/0065_remove_aacu_value_rubric_org_tags.py

```diff
@@ -0,0 +1,16 @@
+from django.db import migrations, models
+
+
+def remove_aacu_value_rubric_org_tags(apps, schema_editor):
+    OrganizationTag = apps.get_model('credo_modules', 'OrganizationTag')
+    OrganizationTag.objects.filter(tag_name='AAC&U VALUE Rubric').delete()
```

Comment:
This statement will work correctly only for the case if: `insights_view=1` + `progress_view=1`. Because for example in case of `insights_view=0` we have to add to the DB N objects for each `AAC&U VALUE Rubric` subtag:
```
OrganizationTag(org='X', tag_name='AAC&U VALUE Rubric - Critical Thinking', insights_view=0).save()
```
As I could see from the current DB table there are no such rows. But anyway can we modify `manage_org_tags` view to remove values from `GROUPED_ORGANIZATION_TAGS` in the begging of POST processing?
I mean something like:
```
def _manage_org_tags_update_data(...):
    OrganizationTag.objects.filter(org='X', tag_name__in=GROUPED_ORGANIZATION_TAGS).delete()
    ...
```

Because will be good to have solution for easly modification any existing tag to support the second level of nesting. If someone asks to add new tag into `GROUPED_ORGANIZATION_TAGS` list it will be too tedious to create new migration for each request.